### PR TITLE
simplify mac build

### PR DESCRIPTION
**Pull request by dwelch-spike**
Removed aws and asbackup dynamic builds. Install cmake via brew.

### CODE REVIEW

**Review by jdogmcsteezy — CHANGES_REQUESTED** [review]
LGTM, but I would change the mac-workflow to build on PR into test-ready in order to test

File: .github/workflows/backup-artifact-linux.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Build:Main
 on:
   push:
-    branches: [ master, test-ready ]
+    branches: [ master ]
```
Comment (jdogmcsteezy) [inline .github/workflows/backup-artifact-linux.yml:4]: you sure you don't want the extra check?

File: .github/workflows/mac-artifact.yml [hunk 2]
```diff
@@ -2,6 +2,8 @@ name: Mac Artifact
 on:
   push:
     branches: [ master, test-ready ]
```
Comment (jdogmcsteezy) [inline .github/workflows/mac-artifact.yml]:
```suggestion
    branches: [ master, test-ready, "bugfix-*", actionsHub ]
```
I find both these useful.  The bugfix one is necessary I think

File: .github/workflows/mac-artifact.yml [hunk 3]
```diff
@@ -42,33 +43,9 @@ jobs:
...
-      if: steps.cache-cmake.outputs.cache-hit != 'true'
-      run: |
-        ./bootstrap 
-        make
-      working-directory: CMake-${{ env.CMAKE_VERSION }}
-    - name: install cmake
-      run: |
-       sudo make install
-      working-directory: CMake-${{ env.CMAKE_VERSION }}
+        brew install openssl@1.1 readline cmake
```
Comment (jdogmcsteezy) [inline .github/workflows/mac-artifact.yml:46]: Do we still need 1.1?

**Review by jdogmcsteezy — APPROVED** [review]
some questions but will approve

File: .github/workflows/backup-artifact-linux.yml (hunk 1 above)
Comment (dwelch-spike) [reply .github/workflows/backup-artifact-linux.yml:4]: This workflow sets the latest tag. I don't want it set to any commits on test-ready.

**Pull request merged**